packages/marketplace/src/resources/locales/fr.json

```diff
@@ -44,24 +44,24 @@
     },
     "devTools": {
       "title": "Outils de développeurs de thèmes",
-      "noThemeInstalled": "(Error: No marketplace theme installed)",
-      "noThemeManifest": "(Error: No theme manifest found)",
-      "colorIniEditor": "(Color.ini Editor)",
+      "noThemeInstalled": "(Erreur: Aucun thème du marché n’est installé)",
+      "noThemeManifest": "(Erreur: Aucun manifeste de thème trouvé)",
+      "colorIniEditor": "(Éditeur Color.ini)",
       "colorIniEditorPlaceholder": "([your-color-scheme-name])",
```

Comment:
This one would be good too if you're able

Reply:
> This one would be good too if you're able

Done 👍🏻